cli/src/commands/squash.rs

```diff
@@ -50,7 +50,8 @@ use crate::ui::Ui;
 ///
 /// If the source became empty and both the source and destination had a
 /// non-empty description, you will be asked for the combined description. If
-/// either was empty, then the other one will be used.
+/// either was empty, then the other one will be used. As an exception, when
+/// `--keep-emptied` is used, each commit keeps its own description.
```

Comment:
An alternative solution is to update the paragraph with "If the source became empty" -> "If the source was abandoned" (or perhaps "If the source was abandoned (because it became empty and `--keep-emptied` was not requested)"). Thoughts?

Reply:
I like that phrasing, I think it's much clearer since there's no special-casing.

Reply:
Agreed.